app/services/spree/product_feed_service.rb

```diff
@@ -4,10 +4,10 @@ class ProductFeedService
 
     attr_reader :product, :variant, :store
 
-    def initialize(variant)
+    def initialize(variant, store = Spree::Store.default)
```

Comment:
not an issue, just curious on the reasoning for adding store as a default argument rather than initializing it to Spree::Store.default within the method. I'd hazard a guess that you anticipate passing this stores other than the default store in future?

Reply:
Previously it was initialized in the method. That meant a new SQL query every time you initialized an object. My goal for this PR was to reduce the volume of SQL queries so this was an easy one to do.